### ISSUE TITLE
the normal training loss profile 

### ISSUE THREAD

**Issue by Wolfybox** [hustvl/MapTR#164]
hi, author.
I have tried training MapTRv2 with the given three auxiliary tasks: depth, pv_seg, bev_seg.
However, after training for 24 epochs, I observed that the loss of these auxiliary tasks only slightly decrease. Their loss curves also show strong fluentuation. 

loss_depth: from 6.8 to 6.64
loss_pv_seg: from 0.265 to 0.215
loss_bev_seg: from 0.138 to 0.118

Note that the depth loss is still large at the end. 

Other losses also look potentially unconverged. 
loss_pts at the last dec layer: from 12 to 4
loss_cls at the last dec layer: 0.65 to 0.35

I wonder whether the above phenomenons are normal for training.

**Comment by JJLi0427** [comment]
I meet same problem, have you slove it? I try to visualize the result, instance pts seem to not learn well
